ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/ApprovedDeferrals.tsx

```diff
@@ -123,6 +123,7 @@ function ApprovedDeferrals() {
                         searchFilter={searchFilter}
                         onFilterChange={(newFilter) => setSearchFilter(newFilter)}
                         searchOptions={searchOptions}
+                        isRegexMatcher={false}
```

Comment:
I noticed you had to add `isRegexMatcher={false}` to all the call sites that where we don't want this used yet.

I'm a proponent of backwards-compatible additions to API/SDK/call signatures, so I suggest that you a default prop value of to `isRegexMatcher` of `false`, so that we don't have to update every call site. This reduces the maintenance overhead, especially if we adopt this new filter in other places in the future.

Reply:
Ironically, I suggested the opposite (based on experience reading code for classic components) so `isRegexMatcher={false}` becomes an intentional decision instead of on oversight.

Reply:
What about option 3? Make this the only behavior of the component in all places it is used. The new, unreleased, exception management pages are the only place this is false, but there was discussion with UX in the past that regex matching for this type of search should be the default product-wide. I can make the update in this PR if there are no objections. CC: @sachaudh @mansursyed

Reply:
@sachaudh According to Slack - we are going all regex by default https://redhat-internal.slack.com/archives/C03JMGWJYMD/p1705678815828799.

I've updated this PR accordingly.